content/english/csharp-guess-the-word/variables.md

```diff
@@ -1,96 +0,0 @@
----
```

Comment:
What happened to this file?

Reply:
Fixed :) I have no idea how that happened